Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/rest/Concepts.java

```diff
@@ -189,10 +222,21 @@ private JsonValidationResult checkJsonObject(JSONObject jsonObject) {
                     false);
         }
 
-        if (jsonObject.get(JsonFields.DESCRIPTION) == null) {
+        String pos = jsonObject.get(JsonFields.POS).toString();
```

Comment:
never mind it, it's tested in the beginning of the method. but you should keep things together that belong together.